### PR TITLE
 qa: fixup quota values exceeding 64 bit signed int value

### PR DESCRIPTION
**Pull request by vshankar**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/workunits/fs/quota/quota.sh [hunk 1]
```diff
@@ -106,8 +106,8 @@ expect_false setfattr -n ceph.quota.max_bytes -v -9223372036854775809 .
 
 setfattr -n ceph.quota.max_bytes -v 0 .
 setfattr -n ceph.quota.max_bytes -v 1Ti .
-setfattr -n ceph.quota.max_bytes -v 8388608Ti .
-expect_false setfattr -n ceph.quota.max_bytes -v 8388609Ti .
+setfattr -n ceph.quota.max_bytes -v 8388607Ti .
+expect_false setfattr -n ceph.quota.max_bytes -v 8388608Ti .
```
Comment (dparmar18) [inline qa/workunits/fs/quota/quota.sh:110]: thanks for fixing this

do you think we should warn users while taking overflowing input?

Comment (vshankar) [reply qa/workunits/fs/quota/quota.sh:110]: warn where? logs? that's already done.

If you are talking about via the CLI itself, maybe have a mention in the docs.

Comment (dparmar18) [reply qa/workunits/fs/quota/quota.sh:110]: add a note in docs warning users not to exceed? looks quite trivial there

Comment (vshankar) [reply qa/workunits/fs/quota/quota.sh:110]: I'm not clear the warning you mention. We do return EINVAL and dump in the logs.

Comment (dparmar18) [reply qa/workunits/fs/quota/quota.sh:110]: return a warning "Integer overflow, please resort to acceptable range", something like this on CLI

Comment (vshankar) [reply qa/workunits/fs/quota/quota.sh:110]: Maybe. Let's do it as a separate change.

**Comment by vshankar** [comment]
@dparmar - https://pulpito.ceph.com/?branch=wip-vshankar-testing-20230309.101043

File: qa/workunits/fs/quota/quota.sh (hunk 1 above)
Comment (dparmar18) [reply qa/workunits/fs/quota/quota.sh:110]: yeah sure, i'll close this conversation. will approve the pr

**Review by dparmar18 — APPROVED** [review]
LGTM

**Review by dparmar18 — APPROVED** [review]

**Comment by vshankar** [comment]
https://tracker.ceph.com/projects/cephfs/wiki/Main#13-Mar-2023

**Comment by dparmar** [comment]
> @dparmar - https://pulpito.ceph.com/?branch=wip-vshankar-testing-20230309.101043

Looks like you mistook me for someone else.

**Pull request merged**